src/frontends/tensorflow/include/openvino/frontend/tensorflow/decoder.hpp

```diff
@@ -19,13 +19,6 @@ class TENSORFLOW_API DecoderBase {
     /// \return Shared pointer to appropriate value converted to openvino data type if it exists, 'nullptr' otherwise
     virtual ov::Any get_attribute(const std::string& name) const = 0;
 
-    /// \brief Get attribute value by name
-    ///
-    /// \param name Attribute name
-    /// \return Shared pointer to appropriate value in native tensorflow data type if it exists,
-    /// 'nullptr' otherwise
-    virtual ov::Any get_native_attribute(const std::string& name) const = 0;
```

Comment:
This functionality returns protobuf structures as is, it can lead to segfalts, which is happened in this case. It is not needed and should be deleted.